cov chart.R

```diff
@@ -1,6 +1,48 @@
 library(tidyverse)
 library(readr)
+
 Confirmed <- read_csv("https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Confirmed.csv")
+Deaths <- read_csv("https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Deaths.csv")
+Recovered <- read_csv("https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Recovered.csv")
+isoalpha3 <- read_csv('https://gist.githubusercontent.com/tadast/8827699/raw/7255fdfbf292c592b75cf5f7a19c16ea59735f74/countries_codes_and_coordinates.csv')%>%
+  select(Country, `Alpha-3 code`)%>%
+  rename(iso_alpha3 = `Alpha-3 code`)
+
+tidy_CSSE <- function(data, value_col = "Confirmed"){
```

Comment:
do we want to do `value_col = "Confirmed"` or would be better to leave this argument blank? or have the value populated based on which of 3 files is being processed?